Header header
geometry_msgs/Vector3 force #supposed applied forces
geometry_msgs/Vector3 torque #supposed applied torques
float64[12] nozzle_openings 

